README.md

```diff
@@ -34,6 +34,21 @@ functions as a "swiss army knife" for machine learning researchers.  In addition
 to its powerful C++ interface, mlpack also provides command-line programs and
 Python bindings.
 
+mlpack uses as [open governance model](./GOVERNANCE.md) and is fiscally
```

Comment:
When reading this a second time the `as` sounds strange, the template uses `an` so I'm wondering if this is correct as well?

Reply:
You're right, not sure how I missed that.  Fixed in 058915d.